Libraries/src/Amazon.Lambda.AspNetCoreServer/AbstractAspNetCoreFunction.cs

```diff
@@ -251,6 +255,46 @@ protected virtual IHostBuilder CreateHostBuilder()
             return builder;
         }
 
+        #if NET8_0_OR_GREATER
+        /// <summary>
+        /// Return one or more <see cref="HttpRequestMessage"/>s that will be used to invoke
+        /// Routes in your lambda function in order to initialize the ASP.NET Core and Lambda pipelines
+        /// during <see cref="SnapshotRestore.RegisterBeforeSnapshot"/>,
+        /// improving the performance gains offered by SnapStart.
+        /// <para />
+        /// The returned <see cref="HttpRequestMessage"/>s must have a relative
+        /// <see cref="HttpRequestMessage.RequestUri"/>.
+        /// <para />.
```

Comment:
Since the PR is using `ReadAsStringAsync` to read the request body and not handling binary request payloads can you add a comment here that this SnapStart warm up feature only support text based request payload.

Reply:
fixed